tutorials/core-concepts/users-and-permissions/managing-accesss-control.md

```diff
@@ -1,5 +1,159 @@
 ---
-title: Manaing Access Control
+title: Managing Access Control
 layout: tutorial
 ---
 
+# Managing Access Control
+
+In most cases, you essentially have three levels of permissions:
+
+* The administrator (`admin`), who can do anything
+
+* The editor or writer (`edit`), who can create content, but doesn't have access to configuration or management of the site
+
+* The guest (`guest`), who can look at all the cool stuff that everyone else makes
+
+Apostrophe makes it easy to set this up with minimal configuration, but if that isn't enough, you have have the option to create custom permissions schemes with more control over exactly who has access to what.
+
+## Standard Permissions
+
+Apostrophe ships with two user groups enabled: `admin` and `guest`. These are configured in the `index.js` for `apostrophe-user`. For some deployments, this might be all you need, but if you need just a little bit more, you can easily add `edit`. To add it, create a new configuration in `app.js` that overrides the default.
```

Comment:
`apostrophe-boilerplate`, which is used as a basis for your project if you use our CLI to create it, ships with...

Reply:
Updated to indicate that the groups come from apostrophe-boilerplate and the CLI, not just Apostrophe